GoogleForCreators/web-stories-wp
# Add foundation for plugin activation notice

## Summary



## Relevant Technical Choices

Notice is rendered in React for consistency and forward compatbility.

## To-do



## User-facing changes

Looks very unspectacular, but is actually rendered via React:

<img width="1054" alt="Screenshot 2020-06-30 at 18 34 40" src="https://user-images.githubusercontent.com/841956/86152478-8260d680-bb00-11ea-95d1-c7d106825984.png">

## Testing Instructions

1. Deactivate Web Stories plugin
1. Activate plugin again
1. See notice

---



See #2853
